src/Security/LogoutForm.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace SilverStripe\Security;
+
+use SilverStripe\Control\Director;
+use SilverStripe\Control\RequestHandler;
+use SilverStripe\Control\Session;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\Form;
+use SilverStripe\Forms\FormAction;
+use SilverStripe\Forms\HiddenField;
+use SilverStripe\Forms\Validator;
+
+/**
+ * Log out form to display to users who arrive at 'Security/logout' without a
+ * CSRF token. It's preferable to link to {@link Security::logout_url()}
+ * directly - we only use a form so that we can preserve the "BackURL" if set
+ */
+class LogoutForm extends Form
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(
+        RequestHandler $controller = null,
+        $name = self::DEFAULT_NAME,
+        FieldList $fields = null,
+        FieldList $actions = null,
+        Validator $validator = null
+    ) {
+        $this->setController($controller);
+
+        $fields = $this->getFormFields();
```

Comment:
I’ve now wrapped this in an `if (!$fields)` conditional, which is what we do for `MemberLoginForm`